.github/workflows/scripts/css_check.py

```
@@ -0,0 +1,362 @@
+        output.append("CSS Import Violations:")
+        for violation in result.violations:
+            output.append(
+                f"- {violation.file_path}: "
+                f"{violation.css_file} ({violation.reason})"
+            )
+        exit_code = 0
+
+    if result.embedded_violations:
+        output.append("\nEmbedded CSS Violations:")
+        for violation in result.embedded_violations:
+            for line_number, css_code in zip(
```

Ruff (0.8.2)

Loop control variable `line_number` not used within loop body

(B007)